tests/integration/components/container-query/splattributes-test.ts

```diff
@@ -10,17 +11,9 @@ module('Integration | Component | container-query', function (hooks) {
   setupContainerQueryTest(hooks);
 
   module('...attributes', function () {
-    test('The component accepts splattributes', async function (assert) {
-      assert.expect(2);
-
-      this.fetchData = () => {
-        assert.ok(
-          true,
-          '{{did-insert}} modifier works. (But we should find a better way to separate concerns!)'
-        );
-      };
```

Comment:
This removes the test being dependent on `@ember/render-modifiers` to have been installed.